lib/vagrant-libvirt/action/create_domain.rb

```diff
@@ -280,10 +280,10 @@ def call(env)
             env[:ui].info(" -- smartcard device:  mode=#{@smartcard_dev[:mode]}, type=#{@smartcard_dev[:type]}")
           end
 
-          @qargs = config.qemu_args
-          if not @qargs.empty?
+          @qemu_args = config.qemu_args
```

Comment:
use different name for variable - it was really hard to debug mess of same name functions and variables.